test/deploy/operator.yaml

```diff
@@ -0,0 +1,60 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: managed-upgrade-operator
+  namespace: test-managed-upgrade-operator
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      name: managed-upgrade-operator
+  template:
+    metadata:
+      labels:
+        name: managed-upgrade-operator
+    spec:
+      serviceAccountName: managed-upgrade-operator
+      affinity:
+        nodeAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            nodeSelectorTerms:
+            - matchExpressions:
+              - key: node-role.kubernetes.io/master
+                operator: In
+                values:
+                - ""
+      tolerations:
+      - effect: NoSchedule
+        key: node-role.kubernetes.io/master
+      - effect: NoExecute
+        key: node-role.kubernetes.io/master
+      containers:
+        - name: managed-upgrade-operator
+          # Replace this with the built image name
+          # This will get replaced on deploy by /hack/generate-operator-bundle.py
+          image: quay.io/app-sre/managed-upgrade-operator:latest
```

Comment:
It is probably worth calling out in the documentation instructions that the person would need to modify this line to point to their personal repo,. as they may not notice the comment in the yaml.